app/controllers/lambda_callback/resolution_proof_result_controller.rb

```diff
@@ -25,8 +25,10 @@ def result_id_parameter
     end
 
     def resolution_result_parameter
-      params.require(:resolution_result).permit(:exception, :success, :timed_out, :transaction_id,
-                                                errors: {}, context: {})
+      params.require(:resolution_result).permit(
+        :exception, :success, :timed_out, :transaction_id,
+        errors: {}, context: {}
+      )
```

Comment:
Just curious, would the following be an _error_?

```suggestion
      params.require(:resolution_result).permit(
        :exception,
        :success,
        :timed_out,
        :transaction_id,
        errors: {},
        context: {},
      )
```

Reply:
I don't take any issues with it as-is. I would maybe consider splitting it up into an array that gets passed into the function (if it accepts it in that form)

```ruby
permitted_params = [:exception, :success, :timed_out, :transaction_id, errors: {}, context: {}]
params.require(:resolution_result).permit(permitted_params)
```